app/graph/types/team_type.rb

```diff
@@ -281,4 +281,12 @@ def shared_teams
   def tipline_messages(uid:)
     TiplineMessagesPagination.new(object.tipline_messages.where(uid: uid).order('sent_at DESC'))
   end
+
+  field :articles, ExplainerType.connection_type, null: true do
+    argument :type, GraphQL::Types::String, required: true
+  end
+
+  def articles(type:)
+    object.explainers if type == 'explainer'
+  end
```

Comment:
This is where we should use the new `ArticleUnion` type mentioned in the respective Jira ticket.